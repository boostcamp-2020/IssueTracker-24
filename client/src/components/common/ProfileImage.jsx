import React, { useContext } from 'react';
import styled from 'styled-components';

const Image = styled.img`
  width: ${(props) => `${props.size}px`};
  height: ${(props) => `${props.size}px`};
  border-radius: ${(props) => `${props.size / 2}px`};
  vertical-align: middle;
`;

const ProfileImage = ({ image, size }) => {
  return <Image src={image} size={size}></Image>;
};

export default ProfileImage;
