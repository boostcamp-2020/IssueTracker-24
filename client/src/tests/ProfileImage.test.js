import React from 'react';
import { render } from '@testing-library/react';
import ProfileImage from '../components/common/ProfileImage';

describe('<ProfileImage/>', () =>{
    const imageSrc = 'jinhyun.png';
    const size='300px';
    it('matches snapshot', () =>{
        const utils = render(<ProfileImage image={imageSrc} size={size} />);
        expect(utils.container).toMatchSnapshot();
    });
})