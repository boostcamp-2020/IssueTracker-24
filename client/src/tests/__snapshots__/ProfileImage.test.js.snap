// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileImage/> matches snapshot 1`] = `
<div>
  <img
    class="sc-bdnylx gIESGv"
    src="jinhyun.png"
  />
</div>
`;
